apps/omg_watcher_rpc/test/omg_watcher_rpc/tracer_test.exs

```diff
@@ -0,0 +1,107 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.TracerTest do
+  @app :omg_watcher_rpc
+
+  use ExUnit.Case, async: true
```

Comment:
your test is fiddling with global state `Application.put_env(@app, :api_mode, :watcher)`.

async means that two processes will execute the two tests concurrently, so tests not failing is luck :D

Reply:
Whoops, let me fix this.